#amplitude: -0.4
#period: 2.0
#repetitions: 10
#sampling_period: 0.001
#controlled_joint: right_j6

amplitude: 0.4
period: 1
repetitions: 5
sampling_period: 0.001
controlled_joint: right_j0
